oauth2_provider/oauth2_validators.py

```diff
@@ -816,8 +816,10 @@ def _load_id_token(self, token):
             return None
         try:
             jwt_token = jwt.JWT(key=key, jwt=token)
-            claims = json.loads(jwt_token.claims)
-            return IDToken.objects.get(jti=claims["jti"])
+            # TODO: Once double write is live, we can read from jti
```

Comment:
@mackosx I think we can link this todo on this ticket?
https://app.shortcut.com/greenspace/story/77179/step-2-remove-id-token-logic-from-1-5-0-grn-custom-release-of-oauthlib-toolkit

Reply:
Good call!